Mathlib/Algebra/Group/UniqueProds.lean

```diff
@@ -169,6 +173,15 @@ theorem mulHom_map_iff (f : G ↪ H) (mul : ∀ x y, f (x * y) = f x * f y) :
 #align unique_mul.mul_hom_map_iff UniqueMul.mulHom_map_iff
 #align unique_add.add_hom_map_iff UniqueAdd.addHom_map_iff
 
+open Finset MulOpposite in
+@[to_additive]
+theorem of_mulOpposite (h : @UniqueMul Gᵐᵒᵖ (MulOpposite.mul G)
+      (B.map ⟨_, op_injective⟩) (A.map ⟨_, op_injective⟩) (op b0) (op a0)) :
```

Comment:
Does this not work?
```suggestion
theorem of_mulOpposite (h : UniqueMul Gᵐᵒᵖ
      (B.map ⟨_, op_injective⟩) (A.map ⟨_, op_injective⟩) (op b0) (op a0)) :
```

Reply:
I am not sure.  I initially had a version with `image`, but then I needed a `DecidableEq` assumption.  You may be right that the `@` was needed in an intermediate version, but not now.  Feel free to push the better version, if it works (I'm on mobile now and won't be able to check until tomorrow).